modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/GroupModelListener.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.model.listener;
+
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.service.ObjectDefinitionLocalService;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+import com.liferay.portal.kernel.exception.ModelListenerException;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.model.BaseModelListener;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.ModelListener;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Jorge García Jiménez
+ */
+@Component(service = ModelListener.class)
+public class GroupModelListener extends BaseModelListener<Group> {
+
+	@Override
+	public void onAfterRemove(Group group) throws ModelListenerException {
+		try {
+			for (ObjectDefinition objectDefinition :
+					_objectDefinitionLocalService.getObjectDefinitions(
+						group.getCompanyId(), true, false,
+						WorkflowConstants.STATUS_APPROVED)) {
+
+				for (ObjectEntry objectEntry :
+						_objectEntryLocalService.getObjectEntries(
+							group.getGroupId(),
+							objectDefinition.getObjectDefinitionId(),
+							QueryUtil.ALL_POS, QueryUtil.ALL_POS)) {
+
+					try {
+						_objectEntryLocalService.deleteObjectEntry(objectEntry);
+					}
+					catch (PortalException portalException) {
+						_log.error(portalException);
+					}
+				}
+			}
+		}
+		catch (PortalException portalException) {
+			_log.error(portalException);
+		}
```

Comment:
@jgarcialr could you please include the following commits from the previous pull request?

https://github.com/brianchandotcom/liferay-portal/commit/e3107c0419c9ef341053253c73fc482ac2e62245 and https://github.com/brianchandotcom/liferay-portal/commit/2398462aaa3ba3c645db8d72ff42dc7dac9fb1bb

Thanks!

Reply:
Oh, I missed those changes.
I'll add those to the PR.